src/systemd-rdma-settings.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
agreed. we don't depend on bash, and maybe it's not a good idea to require user to install bash to use ceph.